### DISCUSSION TITLE
prediction file for evaluation script contains colon

### DISCUSSION THREAD

**Discussion by waskita** [p-lambda/wilds#130] [Ideas]
evaluate.py is very useful for calculating prediction score. One problem though: standard prediction filename for evaluate.py contain colon character (':'), which is invalid character for filename in Windows. 

For example: iwildcam_split:id_test_seed:0_epoch:best_pred.csv (taken from https://wilds.stanford.edu/submit/)

As the result, we cannot run evaluate.py on Windows without some modification.

Suggestion: make sure all filenames can work on popular operating systems (Linux, Mac OSX, Ubuntu, etc)

**Comment by kohpangwei** [comment]
Ah, thanks for pointing that out, and sorry about that! Our Windows users typically run WSL so we haven't run into this issue.
